Saving last configuration file to .certora_internal/24_06_07_11_34_04_296/run.conf
There is no TAC file. Going to script EVMVerifier/certoraBuild.py to main_with_args()
Creating dir /home/shan/my_project/test-audits/.certora_internal/24_06_07_11_34_04_296/.certora_config
In /home/shan/my_project/test-audits/Balances.spec, found the imports: [('./helpers.spec', '1:1'), ('./IOwnable.spec', '2:2')]
In /home/shan/my_project/test-audits/helpers.spec, found the imports: []
In /home/shan/my_project/test-audits/IOwnable.spec, found the imports: []
copying spec file /home/shan/my_project/test-audits/Balances.spec to /home/shan/my_project/test-audits/.certora_internal/24_06_07_11_34_04_296/.certora_sources/.0_Balances.spec.spec
writing /home/shan/my_project/test-audits/.certora_internal/24_06_07_11_34_04_296/.certora_verify.json
Path to typechecker is /home/shan/.local/lib/python3.10/site-packages/certora_jars/Typechecker.jar
running ['java', '-jar', '/home/shan/.local/lib/python3.10/site-packages/certora_jars/Typechecker.jar', '-buildDirectory', '/home/shan/my_project/test-audits/.certora_internal/24_06_07_11_34_04_296']
Failed to compile spec files:
[main] ERROR ALWAYS - Found errors in Balances.spec:
[main] ERROR ALWAYS - Error in spec file (Balances.spec:21:5): IERC20 is not a valid CVL type
[main] ERROR ALWAYS - Error in spec file (Balances.spec:49:5): IERC20 is not a valid CVL type
build failed
Failure traceback: 
CertoraUserInputError: CVL specification syntax check failed
Encountered an error running Certora Prover:
CVL specification syntax check failed
